tools/model-compiler/src/main/java/io/spine/tools/compiler/annotation/MethodNameAnnotator.java

```diff
@@ -59,7 +59,7 @@ public void annotate() {
             SourceVisitor<?> visitor = new AnnotateMethods();
             descriptors().stream()
                          .map(TypeSet::from)
-                         .flatMap(typeSet -> typeSet.types().stream())
+                         .flatMap(typeSet -> typeSet.allTypes().stream())
                          .map(type -> type.javaClassName().resolveFile())
```

Comment:
The more Java8 style will be:
```
descriptors().stream()
    .map(TypeSet::from)
    .map(TypeSet::allTypes)
    .flatMap(ImmutableSet::stream)
    .map(Type::javaClassName)
    .map(ClassName::resolveFile)
    .forEach(file -> rewriteSource(file, visitor));
```

And, IMO it is a bit more readable and easy to understand.